pinc/upload_file.inc

```diff
@@ -0,0 +1,276 @@
+<?php
+include_once($relPath.'misc.inc'); // get_upload_err_msg(), attr_safe(), return_bytes()
+
+// This contains functions for uploading files
```

Comment:
Can you please re-word-wrap this (and really all updated comment blocks) at 80 columns? I think it would be easier to read. It won't always be perfect but lets make use of the full 80-characters when we can.